mintpy/data/colormaps/romanian.cpt

```diff
@@ -0,0 +1,261 @@
+0.709804 233 136 28 0.713725 232 135 28
+0.713725 232 135 28 0.717647 232 133 28
+0.717647 232 133 28 0.721569 232 132 28
+0.721569 232 132 28 0.725490 231 131 29
+0.725490 231 131 29 0.729412 231 129 29
+0.729412 231 129 29 0.733333 230 128 29
+0.733333 230 128 29 0.737255 230 127 29
+0.737255 230 127 29 0.741176 230 125 29
+0.741176 230 125 29 0.745098 229 124 29
+0.745098 229 124 29 0.749020 229 122 29
+0.749020 229 122 29 0.752941 228 121 29
+0.752941 228 121 29 0.756863 228 119 30
+0.756863 228 119 30 0.760784 228 118 30
+0.760784 228 118 30 0.764706 227 116 30
+0.764706 227 116 30 0.768627 227 114 30
+0.768627 227 114 30 0.772549 227 113 30
+0.772549 227 113 30 0.776471 226 111 30
+0.776471 226 111 30 0.780392 226 109 30
+0.780392 226 109 30 0.784314 226 108 31
+0.784314 226 108 31 0.788235 225 106 31
+0.788235 225 106 31 0.792157 225 105 31
+0.792157 225 105 31 0.796078 225 103 31
+0.796078 225 103 31 0.800000 224 101 31
+0.800000 224 101 31 0.803922 224 100 31
+0.803922 224 100 31 0.807843 224 98 31
+0.807843 224 98 31 0.811765 223 96 31
+0.811765 223 96 31 0.815686 223 95 32
+0.815686 223 95 32 0.819608 222 93 32
+0.819608 222 93 32 0.823529 222 91 32
+0.823529 222 91 32 0.827451 222 90 32
+0.827451 222 90 32 0.831373 221 88 32
+0.831373 221 88 32 0.835294 221 86 32
+0.835294 221 86 32 0.839216 221 85 32
+0.839216 221 85 32 0.843137 220 83 33
+0.843137 220 83 33 0.847059 220 82 33
+0.847059 220 82 33 0.850980 220 80 33
+0.850980 220 80 33 0.854902 219 78 33
+0.854902 219 78 33 0.858824 219 77 33
+0.858824 219 77 33 0.862745 219 75 33
+0.862745 219 75 33 0.866667 218 73 33
+0.866667 218 73 33 0.870588 218 72 33
+0.870588 218 72 33 0.874510 218 70 34
+0.874510 218 70 34 0.878431 217 69 34
+0.878431 217 69 34 0.882353 217 67 34
+0.882353 217 67 34 0.886275 217 65 34
+0.886275 217 65 34 0.890196 216 64 34
+0.890196 216 64 34 0.894118 216 62 34
+0.894118 216 62 34 0.898039 215 60 34
+0.898039 215 60 34 0.901961 215 59 35
+0.901961 215 59 35 0.905882 215 57 35
+0.905882 215 57 35 0.909804 214 55 35
+0.909804 214 55 35 0.913725 214 54 35
+0.913725 214 54 35 0.917647 214 52 35
+0.917647 214 52 35 0.921569 213 51 35
+0.921569 213 51 35 0.925490 213 49 35
+0.925490 213 49 35 0.929412 213 47 35
+0.929412 213 47 35 0.933333 212 46 36
+0.933333 212 46 36 0.937255 212 44 36
+0.937255 212 44 36 0.941176 212 42 36
+0.941176 212 42 36 0.945098 211 41 36
+0.945098 211 41 36 0.949020 211 39 36
+0.949020 211 39 36 0.952941 211 37 36
+0.952941 211 37 36 0.956863 210 36 36
+0.956863 210 36 36 0.960784 210 34 37
+0.960784 210 34 37 0.964706 209 32 37
+0.964706 209 32 37 0.968627 209 31 37
+0.968627 209 31 37 0.972549 209 29 37
+0.972549 209 29 37 0.976471 208 28 37
+0.976471 208 28 37 0.980392 208 26 37
+0.980392 208 26 37 0.984314 208 24 37
+0.984314 208 24 37 0.988235 207 23 37
+0.988235 207 23 37 0.992157 207 21 38
+0.992157 207 21 38 0.996078 207 19 38
+0.996078 207 19 38 1.000000 206 18 38
+N 255 255 255
+B 0 44 223
+F 206 18 38
```

Comment:
Could you add this custom colormap name to the documentation, e.g. here (https://github.com/insarlab/MintPy/blob/main/docs/api/colormaps.md#colormaps-from-cpt-city) if it's from cpt-city?